core/validation/schema.py

```diff
@@ -75,6 +75,7 @@ class SchemaError(ValueError):
 
 _PY_TO_NUMPY_TYPES = {
     "str": "string",
+    "list": "list",  # This whole dict will be removed when we refactor out the numpy types.
```

Comment:
is there a ticket for this? Also, can the comment be appended with "TODO:" and moved up to the actual dict definition line so it's harder to miss / ignore